integrations/index.js

```diff
@@ -35,6 +36,7 @@ const integrations = {
   FACEBOOK_PIXEL: FBPixel.default,
   LOTAME: Lotame.default,
   OPTIMIZELY: Optimizely.default,
+  BUGSNAG: Bugsnag.default
```

Comment:
also add a corresponding entry in cname file. 
can you also check after making the change that selective destination for sending events is working fine for bugsnag